packages/playwright/src/mcp/browser/browserContextFactory.ts

```diff
@@ -174,14 +175,19 @@ class PersistentContextFactory implements BrowserContextFactory {
 
     const browserType = playwright[this.config.browser.browserName];
     for (let i = 0; i < 5; i++) {
+      const launchOptions: LaunchOptions = {
+        tracesDir,
+        ...this.config.browser.launchOptions,
+        ...this.config.browser.contextOptions,
+        handleSIGINT: false,
+        handleSIGTERM: false,
+        ignoreDefaultArgs: [
+          '--disable-extensions',
```

Comment:
I believe we pass it to disable side-effects from preinstalled extensions and now this will enable the extensions for everyone with persistent context, is that what we want? Why not let those that want to use extensions pass `ignoreDefaultArgs` or better yet pass the flags as prescribed in https://playwright.dev/docs/chrome-extensions ?

Reply:
`now this will enable the extensions for everyone with persistent context` - this can't re-enable the extensions because these are new profiles created for mcp only, and they did not have extensions.

We want persistent context users to enjoy the experience of a regular browser, so that they could install extensions there.